packages/react-sdk/src/component-utils/image-loaders.ts

```diff
@@ -0,0 +1,48 @@
+import warnOnce from "warn-once";
+import { allSizes, type ImageLoader } from "./image";
+
+/**
+ * Default image loader in case of no loader provided
+ * https://developers.cloudflare.com/images/image-resizing/url-format/
+ **/
+export const cloudflareRelativeImageLoader: ImageLoader = ({
```

Comment:
just realized that normal open source package user will not have cloudflare installed, so they don't need this cloudflare logic to be there at all, what would be a better place for CF specific stuff?